lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -763,7 +763,7 @@ sub _failed_check {
                    name => $column . "_$id",
                    id => $id,
                    options => $selectable_value,
-                   default_blank => 1,
+                   default_blank => ( 1 != @$selectable_value ) ? 1 : 0
```

Comment:
I'm thinking `( 1 != @$selectable_value )` should be enough here (it's a boolean by itself)